sync/syncer.go

```diff
@@ -255,20 +266,21 @@ func (s *Syncer) lastTickedLayer() types.LayerID {
 
 func (s *Syncer) Synchronise() {
 	defer s.syncRoutineWg.Done()
-
+	s.Info("start sincronize")
 	if s.lastTickedLayer() <= 1 { // skip validation for first layer
 		s.With().Info("Not syncing in layer <= 1", log.LayerId(uint64(s.lastTickedLayer())))
-		s.setGossipSynced(true) // fully-synced, make sure we listen to p2p
+		s.setGossipSynced(Done) // fully-synced, make sure we listen to p2p
 		return
 	}
-
+	s.Info("step 1")
```

Comment:
Do we need all this additional logs? step 1, 2,3  and start?

Reply:
debug leftovers